CHANGELOG.md

```diff
@@ -4,34 +4,31 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [2.0.0]
+## [Unreleased]
+
+
+## 2.0.0 - 2018-07-17
 ### Added
 - Password history rule - [#10](https://github.com/owncloud/password_policy/pull/10) [#34](https://github.com/owncloud/password_policy/issues/34) 
 - Password expiration - [#15](https://github.com/owncloud/password_policy/pull/15) [#27](https://github.com/owncloud/password_policy/issues/27) [#31](https://github.com/owncloud/password_policy/issues/31)
 
-### Changed
-### Removed
 ### Fixed
 - Allow empty passwords for public links even when rules are set - [#36](https://github.com/owncloud/password_policy/issues/36)
 - Remove wrong placeholder on special character input field - [#6](https://github.com/owncloud/password_policy/issues/6)
-- Fix typos - [#5](https://github.com/owncloud/password_policy/issues/5)
+- Spelling errors - [#5](https://github.com/owncloud/password_policy/issues/5)
 
 ### Changed
-- Relicensed under GPLv2
-- Starting with 2.0.0
+- Relicensed as GPLv2 starting with 2.0.0
```

Comment:
maybe move "Changed" after "Added" as this is the order it comes in other changelogs and in https://keepachangelog.com/en/1.0.0/